### PR TITLE
Create readme.md

### PR DESCRIPTION
**Pull request by MordodeMaru**
The installation and setup of Weave GitOps to run Openshift requires an explanation. This is my first attemp at describing the process to get Weave GitOps to spin up and setup GitOps runtime in an Openshift cluster.

### CODE REVIEW

**Comment by MordodeMaru** [comment]
Added first draft of the readme to be reviewed

File: readme.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+
+<!-- 
+Solution
+-->
+
+GitOps allows developers to take control of Kubernetes from within Git. Everything is stored in Git which is the single source of truth for the declared state. Weave GitOps will reconcile what's in Git with the target environment running Kubernetes. All the time. Nothing will drift, everything has an audit trail, no changes go through without a Pull Request. That's the beauty of GitOps: Speed with complete control.
+
+Convergent reconciliation – the magic behind GitOps – is promise theory made real for developers. Having your declared state instantiated immediately and converted in the actual state running in production is a bit of a shocker to most. Initially it feels like black magic but you get used to it, becoming trully cloud native is a matter of time.
```
Comment (kingdonb) [inline readme.md:16]: There are a couple of errors in this paragraph:

> declared state instantiated immediately and converted in the actual state running in production

Not sure how to fix this sentence

> trully

File: readme.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+```
+
+<!-- 
+Not sure how to explain this
+-->
+
+Renaming the context name:
+
+```
+❯ kubectx openshift=.
```
Comment (kingdonb) [inline readme.md:53]: I am not clear why this is needed, and `kubectx` isn't listed or mentioned in the prerequisites.

**Review by kingdonb — CHANGES_REQUESTED** [review]
Couple of things I found that I'd recommend fixing

File: readme.md (hunk 1 above)
Comment (MordodeMaru) [reply readme.md:16]: Hi @kingdonb can you point the specific errors you see in that paragraph. I fail to see them. Why would you want to fix the sentence that contains the word _trully_?

File: readme.md (hunk 2 above)
Comment (MordodeMaru) [reply readme.md:53]: @chanwit may know better, it was part of the explanation he did in Slack.

**Pull request merged**